features/testbot/matching.feature

```diff
@@ -792,4 +792,4 @@ Feature: Basic Map Matching
         When I match I should get
           | trace | geometry             | a:distance | a:duration | a:weight | duration |
           | 2345  | 1.00018,1,1.000315,1 | 15.013264  | 1.5        | 1.5      | 1.5      |
-          | 4321  | 1.00027,1,1.000135,1 | 15.013264  | 1.5        | 1.5      | 1.5      |
\ No newline at end of file
+          | 4321  | 1.00027,1,1.000135,1 | 15.013264  | 1.5        | 1.5      | 1.5      |
```

Comment:
Why do these tests change when the PR says it's only optimizing the map-matching performance?

Reply:
It impacts numeric stability a little bit, which can leas to different matchings being chosen.